tests/admin/test_controller.py

```diff
@@ -1174,6 +1183,15 @@ def test_libraries_get_with_multiple_libraries(self):
             eq_(l1.library_registry_shared_secret, libraries[0].get("library_registry_shared_secret"))
             eq_(l2.library_registry_shared_secret, libraries[1].get("library_registry_shared_secret"))
 
+            eq_({}, libraries[0].get("settings"))
+            eq_(3, len(libraries[1].get("settings").keys()))
+            settings = libraries[1].get("settings")
+            eq_("5", settings.get(Configuration.FEATURED_LANE_SIZE))
+            eq_(FacetConstants.ORDER_LAST_UPDATE,
+                settings.get(Configuration.DEFAULT_FACET_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME))
```

Comment:
As per the other branch, ORDER_LAST_UPDATE doesn't really make sense as a value for this field.